pkg/dockerregistry/testutil/util.go

```diff
@@ -266,6 +266,45 @@ func TestNewImageStreamObject(namespace, name, tag, imageName, dockerImageRefere
 	}
 }
 
+// GetFakeImageStreamImageGetHandler returns a reaction function for use
+// with wake os client returning one of given imagestream image objects if found.
```

Comment:
`s/wake/fake/`
one of the given images wrapped in imagestreamimage object if found